apps/tests/aws-runtime/scripts/start-local.js

```diff
@@ -27,12 +27,15 @@ const firstProcess = spawn("node", [eventual, "local"], {
 
 // Handler for IPC messages
 firstProcess.on("message", (message) => {
-  console.log(message);
   if (message === "ready") {
     // Run the second script after receiving the "ready" signal
-    const secondProcess = spawn("pnpm", ["test:local"], {
-      env: envVars,
-    });
+    const secondProcess = spawn(
+      "pnpm",
+      ["test:local", ...process.argv.slice(2)],
+      {
+        env: envVars,
+      }
+    );
```

Comment:
Why second process?

Reply:
I wanted the args into test so I could filter the tests down to a specific test.